kyuubi-server/src/main/scala/org/apache/kyuubi/engine/KubernetesApplicationOperation.scala

```diff
@@ -49,6 +48,7 @@ class KubernetesApplicationOperation extends ApplicationOperation with Logging {
   }
 
   override def isSupported(clusterManager: Option[String]): Boolean = {
+    // TODO add deploy mode to check whether is supported
```

Comment:
IMO we should deny client mode, although this is a kind of temporary regression

Reply:
Here just a TODO notice, put comments here for notify developers.
I'll make a follow up PR to let `JpsApplicationOperator` support client mode.

Reply:
A clear rejection is better than uncontrollable behavior